### ISSUE TITLE
update User Guide

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#209]
**Reported by bernd Dorer on 5 Jul 2008 20:28 UTC**
The user guide of NVDA should be updated. The section 9 should move to section 6.3 so we have section 6 for the NVDA-menu.
We in Germany have folowing new sections
- speech dictionarries
- beeps as an synthesizer
- synth setting ring

**Comment by nvdaTrac** [comment]
**Comment by jteh on 11 Aug 2008 23:11 UTC**
This was partially done for 0.6p2. However, speech dictionaries, speech beeps and the settings ring are still not yet covered.

**Comment by nvdaTrac** [comment]
**Comment by mdcurran on 16 Feb 2009 04:56 UTC**
Replying to [bernd Dorer](ticket:129):
>The section 9 should move to section 6.3 so we have section 6 for the NVDA-menu.
What exactly is section 9? currently 9 in the english user guide is trouble shooting.

**Comment by nvdaTrac** [comment]
**Comment by Bernd on 22 Feb 2009 02:22 UTC**
Replying to [mdcurran](comment:2):
> Replying to [bernd Dorer](ticket:129):
> >The section 9 should move to section 6.3 so we have section 6 for the NVDA-menu.
> What exactly is section 9? currently 9 in the english user guide is trouble shooting.

In the user guide of NVDA 0.6p2 section 9 were the NVDA settings. Now they are corectly in Section 6.3.

I think I can close this ticket because you are updating the user guide by your self shortly before the (pre)Release. Before NVDA 0.6p2 I thought you've forgotten to update the user guide.

I think you write the User guide quite well. Thanks
